README.md

```diff
@@ -44,6 +44,15 @@ script/server
 
 This runs the server on localhost:4000.
 
+### Checking the HTML
+
+To proof the HTML for things like dead links, run:
+
+```
+bundle exec jekyll build
```

Comment:
This could be in `/script/test`

Reply:
It could but it has 10 errors due to private links, so I didn't want to encode it as something with pass / fail semantics.

Reply:
Docs are here if our future selves want to make those 10 errors go away, so we can put this in a test: https://github.com/gjtorikian/html-proofer

Reply:
OK, I found I needed `--check-internal-hash=false` to get a similar result, but the link to https://www.gov.uk/government/publications/guide-to-the-general-data-protection-regulation maybe does need changing?

Reply:
^^^ Note for me: this needs a new issue